sig-contributor-experience/charter.md

```diff
@@ -1,51 +1,45 @@
   - [discuss.kubernetes.io]
-  - [GitHub Management]
+  - [GitHub Project Organizational Management]
   - [Mailing lists] / Google groups for the project as a whole (eg: kubernetes-dev@googlegroups.com) and for individual sigs and wgs where the Chairs have provided us ownership
   - [Slack]
   - [/kubernetescommunity] YouTube channel
   - [Zoom]
-- Establish and staff teams responsible for the administration and moderation of these platforms
-  - Teams must be staffed by trusted contributors spanning time zones, see [moderation] for more detail
-  - They are authorized to take immediate action when dealing with code of conduct issues, see [moderators] for the full list
-  - They are expected to escalate to the project's code of conduct committee when issues rise above the level of simple moderation
-- Work with other SIGs and interested parties in the project to execute GitHub tasks where required, see [GitHub Management] for more detail
-- Own and execute events that are targeted to the Kubernetes contributor community, including:
+  - Perform issue triage on and maintain the [kubernetes/community] repository.  
+  - Maintenance of the Kubernetes Gsuite
+    - Managing @kubernetes.io aliases for project usage
+    - Managing the Kubernetes Community Calendar
   - The weekly [Kubernetes Community meeting]
-  - [Contributor Summit(s)]
-  - [Steering Committee elections] (though we do not own policy creation, see 'out of scope' below)
+  - Content and Format of the [Contributor Summit(s)]
+  - Execution of the [Steering Committee elections]
   - Retrospective moderation for other SIGs upon request
-  - Other events, like other SIG face to face events, upon request and consideration
-- Strategize, build, and execute on scalable [mentoring programs] for all contributor levels. These may include:
-  - [Google Summer of Code]
-  - [Outreachy]
-  - [Meet Our Contributors]
-  - [Group Mentoring - WIP]
-  - [The 1:1 Hour - WIP]
-  - Speed Mentoring sessions at selected KubeCon/CloundNativeCon's
-- Help onboard new and current contributors into the culture, workflow, and CI of our contributor experience through the [contributor guide], other related documentation, [contributor summits] and programs tailored to onboarding.
-- Perform issue triage on and maintain the [kubernetes/community] repository.  
-- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
-- Assist SIGs/WG Chairs and Technical Leads with organizational management operations as laid out in the [sig-governance] doc
-- Distribute contributor related news on various Kubernetes channels, including Cloud Native Compute Foundation ([CNCF]) for posting blogs, social media, and other platforms as needed.
-- Establish and share metrics to measure project health, community health, and general trends, including:
```

Comment:
For your last three comments: It shows as removed in the diff because I shuffled around the order, you have to switch to viewing the whole document to see the new order.